ui/library/dist/index.d.ts

```diff
@@ -312,8 +311,17 @@ declare module '@allenai/pdf-components/src/utils/scroll' {
         * @param pageIndex The index of the page where the position locates at
         * @param leftPoints The horizontal distance between the origin and the position (in PDF coordinates)
         * @param bottomPoints The vertical distance between the origin and the position (in PDF coordinates)
+        * @param rotation The rotation degree of the document
         */
     export function scrollToPosition(pageIndex: number, leftPoints: number, bottomPoints: number, rotation?: PageRotation): void;
+    export function getScrollParent(node: HTMLElement): Nullable<HTMLElement>;
+    export function calculateTopPx({ heightWithMargins, pageIndex, marginTopPixels, height, bottomPixels, }: {
+            heightWithMargins: number;
+            pageIndex: number;
+            marginTopPixels: number;
+            height: number;
+            bottomPixels: number;
+    }): number;
```

Comment:
Decided not to add the unit `px` to these?

Reply:
@pauls-ai2 my bad just add back